app/views/inputBox.view.js

```diff
@@ -1,7 +1,9 @@
 var MainView = Backbone.View.extend({
     el: $("#my-app"),
     initialize: function (options) {
+
         this.collection = new RectanglesCollection([new Rectangle(), new Rectangle()]);
+        this.collection.once('update', this.drawRectangles.bind(this));
         this.collection.on('change', this.drawRectangles.bind(this));

```

Comment:
but you are doing fetch once per one app session? if yes you are don't have to make collection.once('...
and combine in collection.on(